server/db/write-csv-cql.js

```diff
@@ -0,0 +1,50 @@
+const faker = require('faker');
+const fs = require('fs');
+
+const randomInt = function (max) {
+  return Math.ceil(Math.random() * max);
+};
+
+const hundredMil = 100000000;
+const tenMil = 10000000;
+
+const writeUsers = fs.createWriteStream('photo-data-cql.csv');
+writeUsers.write('listingID,id,src,description\n', 'utf8');
+
+// generate 100 million data points
+// randomly assign listing one of the ten mil
+function writeHundredMillionUsers(writer, encoding, callback) {
+  let i = hundredMil;
+  let id = 0;
+  function write() {
+    let ok = true;
+    do {
+      i -= 1;
+      id += 1;
+      if (id % 10000 === 0) {
+        console.log(`Listing ${id} done`);
+      }
+      const src = `https://bnbair.s3-us-west-1.amazonaws.com/${randomInt(100)}.jpg`;
+      const description = `${faker.lorem.sentence()}`;
+      const listingID = `${randomInt(tenMil)}`;
```

Comment:
I'm not too clear on the difference between listingID and id. listingID refers to the image number and id is what is typed into the url? Since you are randomly generating the listingID, some images are used more than once, right?

Reply:
Oh I see, "listingID" refers to which listing each photo belongs to, and "id" refers to the id of each photo within the photo table. Do you believe it would make more sense to change "id" to "photoID"?

Reply:
While that'd definitely be more clear, that would require you to re-seed right? As long as you are keeping track of it, I don't think a change is 100% necessary